docs/SOPs/Managed_access/Data_Contributor_Agreement_SOP.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Data Contributor Agreement SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 2
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Data Contributor Agreement SOP
+
+This document explains how a wrangler registers a managed access project and helps fill out the data contributor agreement (DCA). See [here](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/Managed_access/Data_Contributor_Agreement_explained.html) if you are unfamiliar with the DCA.
+
+## Create a project in HCA Data Repository Ingest Service 
+This step begins once the wranglers receive a submission request in the form of an email from the contributor. The contributor needs to provide the name and email address for the DCA signers.
+
+1. Register a new project in ingest with minimal details. The only metadata needed is:
+   1. Primary contact/data contributor information 
+   2. Project title. If a title has been provided make sure there is no identifiable information in the project title. If no title was provided use the contributor name and amend with the title provided in the DCA once it’s signed.
+   3. Data use restriction. Fill in the data_use_restriction as GRU-NCU, as the stricter of the Managed access options. 
+   4. Make sure the project is not displayed on the catalogue.
+2. Save the project to generate a project UUID. 
+3. Forward the singer’s contact information to the HCA executive office so they can start the signing process
```

Comment:
replace 'singer's' with 'signing contributor's'

replace 'signing process' with 'DCA signing process'

Reply:
the signers are the PI and institutional legal representative so they aren't all contributors

Reply:
in that case, 

Forward the signing parties' (PI and institutional legal representative) contact information to the HCA executive office so they can start the signing process